lib/configuration.js

```diff
@@ -21,6 +21,11 @@ function coerceToBoolean(value) {
 }
 
 
+function unique(array) {
+  return Array.from(new Set(array));
+}
```

Comment:
This function is intentionally here and in the `Dredd.js` as these files are to be further refactored and I don't want to introduce more dependencies between them or any "utility functions" file just because of this.